m4_sessions/1706-inning/group_six.md

```diff
@@ -1,9 +1,41 @@
 # Group Six
 
-### Session Title:
+### Session Title: 
+#### Code Schools (working title: we'll create something saucier)
 
 ### Session Outcomes: 
+-Reaffirming that we're in the right place: Turing is hard and we all have disparate backgrounds, but we're on the right track for us
+-Insight into the direction of coding/bootcamps: how did we get here and where is this trend going?
+-Be able to identify efficiencies and defficiencies of coding school alums vs CS degree holders
 
 ### Session Outline:
```

Comment:
You are in a good starting place with this outline.  You are also taking on a LOT with the session outcomes.  Be sure to mix direct instruction/lecture with the opportunity for them to discuss.  Spend time getting some resources together to ensure you have succinct teaching points or succinct information on the history of code schools.